Classification/Core/src/main/java/org/tribuo/classification/evaluation/LabelConfusionMatrix.java

```diff
@@ -199,21 +206,34 @@ private int getIDOrThrow(Label key) {
 
     /**
      * Sets the label order used in {@link #toString}.
-     * @param labelOrder The label order to use.
+     *
+     * @param newLabelOrder The label order to use.
      */
-    public void setLabelOrder(List<Label> labelOrder) {
-        this.labelOrder = labelOrder;
+    @Override
+    public void setLabelOrder(List<Label> newLabelOrder) {
+        if (newLabelOrder == null || newLabelOrder.isEmpty()) {
```

Comment:
Do we know what size the labelset should be at this point? Can we easily check for that too while we're at it?

Reply:
This actually allows you to reduce the set of labels that you print (and was designed that way many years ago), if you only care about showing a subset of them. However that's not properly documented, nor do I have a test for it, so I should add one.